proxy/grpcproxy/watcher_single.go

```diff
@@ -53,9 +52,8 @@ func (ws watcherSingle) run() {
 	defer close(ws.donec)
 
 	for wr := range ws.ch {
-		ws.rev = wr.Header.Revision
+		ws.lastSeenRev = wr.Header.Revision
```

Comment:
OK, I see. Can this be renamed to `lastStoreRev`? It hasn't necessarily seen an event on that revision.